src/components/ColorSwatch.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import InlineCode from '../components/markdown-render/inlineCode';
+import theme from 'prism-react-renderer/themes/github';
+
+const ColorSwatch = ({ name, description, hexValue, rgbValue, sassVar, swatchHeight }) => (
+  <div className="sprk-c-Card sprk-o-Stack">
+    <div className="sprk-o-Stack__item">
+      <div
+        style={{
+          minHeight: swatchHeight || '140px',
+          backgroundColor: rgbValue
+        }}
+        className="sprk-c-Card__media"></div>
+    </div>
+
+    <div className="
+      sprk-o-Stack__item
+      sprk-c-Card__content
+      sprk-o-Stack
+      sprk-o-Stack--large
+      "
+    >
+      <h3 className="sprk-b-TypeDisplayFive sprk-o-Stack__item">
+        {name}
+      </h3>
+
+      <p className="sprk-b-TypeBodyTwo sprk-o-Stack__item">
+        {description}
+      </p>
```

Comment:
I don't think this is providing any info that we're not already getting from the name. I think we can delete this.